fuse_constraints/src/absolute_pose_2d_stamped_constraint.cpp

```diff
@@ -69,8 +102,8 @@ fuse_core::Constraint::UniquePtr AbsolutePose2DStampedConstraint::clone() const
 
 ceres::CostFunction* AbsolutePose2DStampedConstraint::costFunction() const
 {
-  return new ceres::AutoDiffCostFunction<NormalPriorPose2DCostFunctor, 3, 2, 1>(
-    new NormalPriorPose2DCostFunctor(sqrt_information_, mean_));
+  return new ceres::AutoDiffCostFunction<NormalPriorPose2DCostFunctor, ceres::DYNAMIC, 2, 1>(
+    new NormalPriorPose2DCostFunctor(sqrt_information_, mean_), total_indices_);
```

Comment:
```suggestion
    new NormalPriorPose2DCostFunctor(sqrt_information_, mean_), sqrt_information_.rows());
```
Then you could drop the `total_indices_` variable from the class state.